integration/business-process-tests/tests-integration/bpmn/src/test/resources/smokeTestng.xml

```diff
@@ -11,14 +11,15 @@
   ~ Unless required by applicable law or agreed to in writing,
   ~ software distributed under the License is distributed on an
   ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  ~ KIND, either express or implied. See the License for the
+  ~ KIND, either express or implied.  See the License for the
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
 
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="mb-suite-initializer">
+<suite name="BPS HumanTask TestSuite" parallel="false">
```

Comment:
Is it good to have space in the suite name? We have used dashes instead of spaces in other places.

Reply:
done